page-resources.hbs

```diff
@@ -41,14 +41,14 @@
         
     <div id="mission" data-aos="fade-up" class="section-1">
 
-        <div class="text-heading" > <h2 style="margin-top: 100px;">Posters</h2></div>
+        <div class="text-heading text-upper" > <h2> RESOURCES </h2></div>

```

Comment:
It’s redundant to have `.text-upper` and a capitalised string inside the `h2`, right? And above on `page-contact` :nerd

Reply:
Yeah, removed now. Thanks.